UI testing for Sat6.8 failed due to organization fields changed

UI testing failed due to the error below:

```
@fixture(scope='module')
    def module_org():
        """Shares the same organization for all tests in specific test module.
        Returns 'Default Organization' by default, override this fixture on
    
        :rtype: :class:`nailgun.entities.Organization`
        """
        default_org_id = (
            nailgun.entities.Organization()
            .search(query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0]
            .id
        )
>       return nailgun.entities.Organization(id=default_org_id).read()

robottelo/tests/foreman/virtwho/conftest.py:28: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.7/site-packages/nailgun/entities.py:5250: in read
    return super(Organization, self).read(entity, attrs, ignore, params)
/usr/local/lib/python3.7/site-packages/nailgun/entity_mixins.py:800: in read
    in _get_entity_ids(field_name, attrs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

field_name = 'config_template'
attrs = {'ancestry': None, 'compute_resources': [], 'created_at': '2020-04-30 10:59:51 UTC', 'default_content_view_id': 1, ...}

    def _get_entity_ids(field_name, attrs):
        """Find the IDs for a one to many relationship.
    
        The server may return JSON data in the following forms for a
        :class:`nailgun.entity_fields.OneToManyField`::
    
            'user': [{'id': 1, …}, {'id': 42, …}]
            'users': [{'id': 1, …}, {'id': 42, …}]
            'user_ids': [1, 42]
    
        Search ``attrs`` for a one to many ``field_name`` and return its ID.
    
        :param field_name: A string. The name of a field.
        :param attrs: A dict. A JSON payload as returned from a server.
        :returns: An iterable of entity IDs.
    
        """
        field_name_ids = field_name + '_ids'
        plural_field_name = pluralize(field_name)
        if field_name_ids in attrs:
            return attrs[field_name_ids]
        elif field_name in attrs:
            return [entity['id'] for entity in attrs[field_name]]
        elif plural_field_name in attrs:
            return [entity['id'] for entity in attrs[plural_field_name]]
        else:
            raise MissingValueError(
>               f'Cannot find a value for the "{field_name}" field. '
                f'Searched for keys named {field_name_ids}, {field_name}, {plural_field_name} '
                f'but available keys are {attrs.keys()}.'
            )
E           nailgun.entity_mixins.MissingValueError: Cannot find a value for the "config_template" field. Searched for keys named config_template_ids, config_template, config_templates but available keys are dict_keys(['label', 'redhat_repository_url', 'system_purposes', 'service_levels', 'service_level', 'select_all_types', 'description', 'created_at', 'updated_at', 'ancestry', 'parent_id', 'parent_name', 'id', 'name', 'title', 'users', 'smart_proxies', 'subnets', 'compute_resources', 'media', 'ptables', 'provisioning_templates', 'domains', 'realms', 'environments', 'hostgroups', 'locations', 'hosts_count', 'parameters', 'simple_content_access', 'owner_details', 'default_content_view_id', 'library_id']).

/usr/local/lib/python3.7/site-packages/nailgun/entity_mixins.py:277: MissingValueError
```

Seems the [Organization fields](https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L5156) missing the ''config_template'' is the root cause.

--- a/nailgun/entities.py
+++ b/nailgun/entities.py
@@ -4873,6 +4873,7 @@ class Location(
         if ignore is None:
             ignore = set()
         ignore.add('realm')
+        ignore.add('config_template')
         return super(Location, self).read(entity, attrs, ignore, params)
 
     def update(self, fields=None):
@@ -5247,6 +5248,7 @@ class Organization(
         if ignore is None:
             ignore = set()
         ignore.add('realm')
+        ignore.add('config_template')
         return super(Organization, self).read(entity, attrs, ignore, params)
 
     def update(self, fields=None):

--- a/nailgun/entities.py
+++ b/nailgun/entities.py
@@ -4818,7 +4818,6 @@ class Location(
         self._fields = {
             'compute_resource': entity_fields.OneToManyField(
                 AbstractComputeResource),
-            'config_template': entity_fields.OneToManyField(ConfigTemplate),
             'description': entity_fields.StringField(),
             'domain': entity_fields.OneToManyField(Domain),
             'environment': entity_fields.OneToManyField(Environment),
@@ -4873,7 +4872,6 @@ class Location(
         if ignore is None:
             ignore = set()
         ignore.add('realm')
-        ignore.add('config_template')
         return super(Location, self).read(entity, attrs, ignore, params)
 
     def update(self, fields=None):
@@ -5154,7 +5152,6 @@ class Organization(
             'compute_resource': entity_fields.OneToManyField(
                 AbstractComputeResource
             ),
-            'config_template': entity_fields.OneToManyField(ConfigTemplate),
             'description': entity_fields.StringField(),
             'domain': entity_fields.OneToManyField(Domain),
             'environment': entity_fields.OneToManyField(Environment),
@@ -5248,7 +5245,6 @@ class Organization(
         if ignore is None:
             ignore = set()
         ignore.add('realm')
-        ignore.add('config_template')
         return super(Organization, self).read(entity, attrs, ignore, params)
 
     def update(self, fields=None):

